pretixprint/app/src/main/java/eu/pretix/pretixprint/connections/System.kt

```diff
@@ -0,0 +1,104 @@
+package eu.pretix.pretixprint.connections
+
+import android.content.Context
+import android.os.Bundle
+import android.os.CancellationSignal
+import android.os.ParcelFileDescriptor
+import android.print.*
+import android.print.PrintAttributes.*
+import com.lowagie.text.pdf.PdfReader
+import eu.pretix.pretixprint.R
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+
+class SystemConnection : ConnectionType {
+    override val identifier = "system"
+    override val nameResource =  R.string.connection_type_system
+    override val inputType = ConnectionType.Input.PDF
+
+    override fun allowedForUsecase(type: String): Boolean {
+        return true
```

Comment:
probably not for receipts? we have no way of rendering them graphically right now